src/install/frontend.py

```diff
@@ -174,13 +95,23 @@ def _install_docker_images(radare):
 
 
 def main(skip_docker, radare, nginx, distribution):
+    apt_packages_path = INSTALL_DIR / 'apt-pkgs-frontend.txt'
+    dnf_packages_path = INSTALL_DIR / 'dnf-pkgs-frontend.txt'
+
+    if distribution != 'fedora':
+        pkgs = read_package_list_from_file(apt_packages_path)
+        apt_install_packages(*pkgs)
+    else:
+        pkgs = read_package_list_from_file(dnf_packages_path)
+        dnf_install_packages(*pkgs)
```

Comment:
Hmm, also it is so simple that imo we don't need a function to do this for us.

Reply:
This has nothing to do with simplicity. It is one of the Clean Code principles: "Don't Repeat Yourself". It is about maintenance: If we need to change this part (e.g. because we add or remove a distro/package manager) we need to edit all three occurrences of the code. If we put it in a function, we instead only need to change it once.

pylint also complains:
```
install/frontend.py:1: [duplicate-code:] Similar lines in 2 files
==install.common:[44:51]
==install.frontend:[100:109]
    if distribution != 'fedora':
        pkgs = read_package_list_from_file(apt_packages_path)
        apt_install_packages(*pkgs)
    else:
        pkgs = read_package_list_from_file(dnf_packages_path)
        dnf_install_packages(*pkgs)

    # flask-security is not maintained anymore and replaced by flask-security-too.
    # Since python package naming conflicts are not resolved automatically, we remove flask-security manually.
```

Maybe we can at least agree to this change to reduce the number of consecutive identical lines:
```suggestion
    if distribution != 'fedora':
        pkgs = read_package_list_from_file(INSTALL_DIR / 'apt-pkgs-frontend.txt')
        apt_install_packages(*pkgs)
    else:
        pkgs = read_package_list_from_file(INSTALL_DIR / 'dnf-pkgs-frontend.txt')
        dnf_install_packages(*pkgs)
```